resources/views/team/show.blade.php

```diff
@@ -0,0 +1,62 @@
+@extends('layouts.app')
+
+@section('content')
+
+<div class="container">
+    <div class="row justify-content-center">
+        <div class="col-md-12">
+            @role('admin')
```

Comment:
Change from `@role('admin')` to `@can('treinador')`

Remember that you also need to change the `@endrole` to `@endcan`